pf/tfgen/not_supported.go

```diff
@@ -33,45 +36,77 @@ func notSupported(sink diag.Sink, prov tfbridge.ProviderInfo) error {
 		})
 	}
 
-	u := &notSupportedUtil{sink}
+	u := &notSupportedUtil{sink: sink}
 
-	if prov.P != nil {
+	skipResource := func(tfToken string) bool { return false }
+	skipDataSource := func(tfToken string) bool { return false }
+	muxedProvider := false
+	if mixed, ok := prov.P.(*muxer.ProviderShim); ok {
+		not := func(f func(string) bool) func(string) bool {
+			return func(s string) bool {
+				return !f(s)
+			}
+		}
+
+		skipResource = not(mixed.ResourceIsPF)
+		skipDataSource = not(mixed.DataSourceIsPF)
+		muxedProvider = true
+	} else if prov.P != nil {
 		u.warn("ProviderInfo.P should be nil for Plugin Framework based providers, populate NewProvider instead")
 	}
 
 	if prov.Resources != nil {
 		for path, res := range prov.Resources {
+			if skipResource(path) {
+				continue
+			}
 			u.resource("resource:"+path, res)
 		}
 	}
 
 	if prov.DataSources != nil {
 		for path, ds := range prov.DataSources {
+			if skipDataSource(path) {
+				continue
+			}
 			u.datasource("datasource:"+path, ds)
 		}
 	}
 
-	if prov.Config != nil {
-		for path, ds := range prov.Config {
-			u.schema("config:"+path, ds)
+	// It might be reasonable to set global values that PF will ignore if this is a
+	// muxed provider, and the SDK side will pick it up.
+	if !muxedProvider {
+		if prov.Config != nil {
+			for path, ds := range prov.Config {
+				u.schema("config:"+path, ds)
+			}
 		}
+
+		u.assertIsZero("PreConfigureCallback", prov.PreConfigureCallback)
+		u.assertIsZero("PreConfigureCallbackWithLogger", prov.PreConfigureCallbackWithLogger)
 	}
 
-	u.assertNotZero("PreConfigureCallback", prov.PreConfigureCallback)
-	u.assertNotZero("PreConfigureCallbackWithLogger", prov.PreConfigureCallbackWithLogger)
+	if len(u.autoNamedResources) > 0 {
+		sort.Strings(u.autoNamedResources)
+		u.warn("SetAutonaming call is currently ignored for bridged resources built with the "+
+			"Plugin Framework. Supporting this feature is tracked in pulumi/pulumi-terraform-bridge#917.\n"+
+			"These resources employ autonaming:\n- %s", strings.Join(u.autoNamedResources, "\n- "))
```

Comment:
"To avoid this warning, exclude these resources from auto naming, for example by adding them to ProviderInfo.Resources after the SetAutonaming call."